hetu-docs/en/admin/properties.md

```diff
@@ -387,9 +387,9 @@ Exchanges transfer data between openLooKeng nodes for different stages of a quer
 ### `exchange.max-retry-count`
 
 > -   **Type:** `integer`
-> -   **Default value:** `10`
+> -   **Default value:** `100`
 >
-> The maximum number of retry for failed task performed by the coordinator before considering it as a permanent failure. This property is used only when exchange.is-timeout-failure-detection-enabled is set to false. This value needs to be atleast 3 (minimum retry count) to take effect.
+> The maximum number of retry for failed task performed by the coordinator before consulting the failure detector module about the remote node status. If the remote node status is GONE or UNRESPONSIVE or UNKNOWN, it is considered as a permanent failure. The actual retry count when query fails may be little more than this parameter. This property is used only when exchange.is-timeout-failure-detection-enabled is set to false. This value needs to be atleast 100 to take effect.
```

Comment:
These are internal states, do we need to mention this? 

The actual retry count when query fails may be little more than this parameter =>
This parameter is the minimum count which is required to decide, not necessary the exact count.  Based on the cluster size, load on the cluster the exact count may vary slightly.